src/main/java/sonardenemeleri/sonar/App.java

```diff
@@ -19,7 +19,10 @@ public static void main( String[] args )
 BigDecimal bd1 = new BigDecimal(d); // Noncompliant; see comment above
 BigDecimal bd2 = new BigDecimal(1.1); // Noncompliant; same result
 
-
+BigDecimal bd3 = new BigDecimal(d); // Noncompliant; see comment above
+BigDecimal bd4 = new BigDecimal(1.1); // Noncompliant; same result
+BigDecimal bd5 = new BigDecimal(1.1); // Noncompliant; same result
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "bd5". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "bd5" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1481)